saleor/graphql/attribute/utils.py

```diff
@@ -143,14 +143,14 @@ def _resolve_attribute_global_id(cls, error_class, global_id: str) -> int:
     def _pre_save_values(
         cls, attribute: attribute_models.Attribute, attr_values: AttrValuesInput
     ):
-        """Lazy-retrieve or create the database objects from the supplied raw values."""
-        get_or_create = attribute.values.get_or_create
+        """Lazy-update or create the database objects from the supplied raw values."""
+        update_or_create = attribute.values.update_or_create
 
         if not attr_values.values:
             return tuple()
 
         return tuple(
-            get_or_create(
+            update_or_create(
```

Comment:
Shouldn't we use `_update_or_create_value` here? I see it has some differences regarding the slug generation, compared to `_pre_save_values`, cc @IKarbowiak?

Reply:
We cannot update the attribute value for attributes with choices, so any of these solutions is incorrect.